config/default.conf

```diff
@@ -9,7 +9,7 @@ ip = "0.0.0.0"
 max_post_body_size = 3000000
 
 # host resolver
-default_resolver_id = "copyrighthub.org"
+default_resolver_id = "openpermissions.org"
```

Comment:
If we change this then all our hubkeys going forwards will have openpermissions.org rather than copyrighthub.org. Is that what we expect to have?